FineTuning_v1.ipynb

```diff
@@ -28,21 +28,91 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "id": "61ea2e95-6d9d-4068-ab98-8cf94bc4d9d0",
    "metadata": {},
    "outputs": [],
    "source": [
     "from datetime import datetime, timedelta\n",
-    "import kaskada as kt\n",
+    "import json\n",
+    "import kaskada as kd\n",
     "import pandas\n",
     "import openai\n",
     "import getpass\n",
     "import pyarrow\n",
     "import datetime\n",
     "\n",
     "# Initialize Kaskada with a local execution context.\n",
-    "kt.init_session()"
+    "kd.init_session()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "53a119b0",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "## Combine Historical Files\n",
+    "\n",
+    "Historical slack messages can be exported by following the instructions in Slack's [Export your workspace data](https://slack.com/help/articles/201658943-Export-your-workspace-data) web page. We'll use these messages to teach BeepGPT about the members of your workspace.\n",
+    "\n",
+    "The export from Slack contains a zip of numererous folders and files. After uncompressing the archive, there are folders for each public channel in your Slack workspace. Inside each folder are json files for each day, which each contain all the events from the day.\n",
+    "\n",
+    "We execute a short python script (utilizing pandas), to concatenate all the data files together into a single parquet file.\n",
+    "\n",
+    "Parquet files store data in columns instead of rows. Some benefits of Parquet include:\n",
+    "* Fast queries that can fetch specific column values without reading full row data\n",
+    "* Highly efficient column-wise compression"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
```

Comment:
Leaving the results intentionally so people can see the shape of the data without running it.